lighthouse-core/test/lib/emulation-test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const emulation = require('../../lib/emulation.js');
+const {getMockedEmulationDriver} = require('../gather/gather-runner-test.js');
```

Comment:
this is within the scope to refactor when the other mock test utils lands, right?

Reply:
oh yeah. i'm dying to do that.

Reply:
yeah that's gonna be great :D always weird to have test files export junk

Reply:
done